docs/en/Audit-Logging.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: global
+title: Audit Logging
+group: Features
+priority: 1
+---
+
+* Table of Contents
+{:toc}
+
+## Overview
+Alluxio supports audit logging of user access to your data. With audit logging, system administrators can
+keep track of each user's access to data in Alluxio.
+
+## Enable Audit Logging
+To enable Alluxio audit logging, you need to set the JVM property. In `./conf/alluxio-env.sh`, add the following two lines:
```

Comment:
... you need to set the JVM property `alluxio.master.audit.logging.enabled`.

Reply:
Done.